tests/end_to_end/tests.py

```diff
@@ -1427,6 +1436,7 @@ def test_run_configuration(self):
 
         RenewCertificate(self).test_run_configuration()
         LocalGroupTest(self).test_run_configuration()
+        self.step_make_owner()
```

Comment:
It would be less confusing if the step name would also include "client", e.g step_client_make_owner